modules/conversion-center/module.php

```diff
@@ -96,6 +105,50 @@ public function __construct() {
 		add_action( 'wp_ajax_elementor_send_clicks', [ $this, 'handle_click_tracking' ] );
 		add_action( 'wp_ajax_nopriv_elementor_send_clicks', [ $this, 'handle_click_tracking' ] );
 
+		if ( ! static::is_pro_active() ) {
+			add_action( 'wp_footer', function () {
+				$query = new \WP_Query( [
+					'post_type' => ConversionCenterModule::CPT_CONTACT_PAGES,
+					'posts_per_page' => -1,
+					'post_status' => 'publish',
+					'fields' => 'ids',
+					'meta_key' => '_elementor_conditions',
+					'meta_compare' => 'EXISTS',
+				] );
+
+				if ( ! $query->have_posts() ) {
+					return;
+				}
+
+				foreach ( $query->posts as $post_id ) {
+					$conditions = get_post_meta( $post_id, '_elementor_conditions', true );
+					if ( ! $conditions ) {
+						continue;
+					}
+					if ( in_array( 'include/general', $conditions ) ) {
+						$document = Plugin::$instance->documents->get( $post_id );
+						$document->print_content();
+						break;
+					}
+				}
+
+			} );
+		}
+
+		add_filter( 'elementor-pro/theme-builder/conditions-manager/additional-cpts', function( array $cpts ) {
+			$cpts[] = self::CPT_CONTACT_PAGES;
+
+			return $cpts;
+		} );
+
+		add_filter( 'elementor-pro/theme-builder/allowed-documents', function( $allowed, $document ) {
```

Comment:
I'm not sure about these filters yet. I don't really understand why you need them here in Core if you're just going to print the content in `wp_footer`.

Reply:
@louiswol94 those filters are needed because I need to pass the contact buttons to enable display conditions in pro. Display conditions depend on extending a class which only exists in pro. I can't use the contact button class in pro.

Reply:
@nicoladj77 Just thinking here...

- Can we run the "register document" conditionally in Core? Only register the document in Core if Pro does not exist, otherwise register the Document in Pro?